pyproject.toml

```diff
@@ -57,6 +61,7 @@ test = [
     "httpx>=0.27.2",
     "pytest-mock>=3.14.0",
     "pytest-recording>=0.13.2",
+    "asgi-lifespan>=2.1.0",
```

Comment:
is that used somewhere?

Reply:
It was at some point but I wound up not needing it.